<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>typeOrder</class>
 <widget class="QWidget" name="typeOrder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>422</width>
    <height>621</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>422</width>
    <height>621</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>422</width>
    <height>621</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tip Sorgu</string>
  </property>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>401</width>
     <height>421</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>301</width>
     <height>143</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="radTonnage">
        <property name="text">
         <string>Minimum Tonaj</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radHull">
        <property name="text">
         <string>Minimum Gövde Boyu</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QSlider" name="slider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item alignment="Qt::AlignHCenter">
       <widget class="QPushButton" name="btnOrder">
        <property name="maximumSize">
         <size>
          <width>129</width>
          <height>23</height>
         </size>
        </property>
        <property name="text">
         <string>Sırala</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelTonnage">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelHull">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
